micmac_li3ds/api.py

```diff
@@ -7,6 +7,14 @@
 class Api(object):
 
     def __init__(self, api_url, api_key):
+        if not api_url:
+            err = 'Error: no api url provided'
+            raise ValueError(err)
+
+        if not api_key:
+            err = 'Error: no api key provided'
+            raise ValueError(err)
+
```

Comment:
I would like to have the api object do the staging if url or key is not provided, so it does not make sense just now but it will :)